admin_app/src/components/ContentCard.tsx

```diff
@@ -32,20 +31,9 @@ export const ContentCard: React.FC<ContentCardProps> = (
       <div className="absolute flex top-2 right-2 ">
         {card.showEditButton ? (
           <>
-            <PencilIcon
-              data-modal-target="edit-content-modal"
-              data-modal-toggle="edit-content-modal"
-              className="w-3 h-3 text-gray-500 cursor-pointer hover:text-red-200"
-              onClick={() =>
-                card.editMe({
-                  content_id: card.content_id,
-                  content_text: card.content_text,
-                })
-              }
-            />
             <TrashIcon
-              className="w-3 h-3 text-gray-500 cursor-pointer hover:text-red-200"
-              onClick={() => card.deleteMe(card.content_id)}
+              onClick={(e) => card.deleteMe(e, card.content_id)}
+              className="h-6 w-6 text-gray-400 hover:text-red-500 cursor-pointer"
```

Comment:
Personally I don't like the large icon but that could be just me. Want to see what others think of it?
Also, we could swap out TrashIcon with something else less ugly.

Reply:
Also we should add more padding to the right of the text so it doesn't get so close to the icon. Last two ideas: (a) maybe the edit and delete icons should show up after you enlarge it (b) maybe the should show up only when you hover over the card

![image](https://github.com/IDinsight/aaq-core/assets/14125957/79a5e1ac-18b0-425f-a047-ec0931548a1a)

Reply:
Reverted to original but but kept edit near the bottom to make clicking easier